lib/huginn_bigcommerce_order_agent/bigcommerce_order_agent.rb

```diff
@@ -153,6 +153,7 @@ def handle(event)
         create_event payload: new_event.merge(
           status: faraday_error.response[:status],
           scope: e.scope,
+          message: e.message,
           response: faraday_error.response[:body],
           request_data: e.data,
```

Comment:
This needs to create an event payload from the API error instance we created. Look at how the product agent is doing it and have this do the same thing.

Reply:
@mrogers-5s Like this?